includes/functions/functions_general_shared.php

```diff
@@ -389,8 +389,11 @@ function request()
     return \Zencart\Request\Request::getInstance();
 }
 
-function zen_updated_by_admin($admin_id = null)
+function zen_updated_by_admin($admin_id = null): string
 {
+    if (empty($admin_id) && empty($_SESSION['admin_id'])) {
+        return '';
```

Comment:
Should this be `return null; `  as is done on line 416?

Reply:
No, because this function returns a string, and the `zen_update_orders_history()` function that calls it always expects a string.